libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -910,6 +910,12 @@ bool _IotMqtt_IncrementConnectionReferences( _mqttConnection_t * pMqttConnection
     if( disconnected == false )
     {
         ( pMqttConnection->references )++;
+
+        /* In some implementations IotLog() maps to C standard printing API 
+         * that need specific primitive types for format specifiers. Also, 
+         * inttypes.h has may not available on some C99 compilers, despite 
```

Comment:
Need to fix this grammar – "Also, inttypes.h may not be available on some C99 compilers..."